examples/advanced_pytorch/README.md

```diff
@@ -2,7 +2,6 @@
 
 This example demonstrates an advanced federated learning setup using Flower with PyTorch. It differs from the quickstart example in the following ways:
 
-- 10 clients (instead of just 2)
 - Each client holds a local dataset of 5000 training examples and 1000 test examples
```

Comment:
@charlesbvll is this true when passing the `--toy` option? Can you make a remark about it?

Reply:
Technically it is still true that each client holds that much data, but indeed when running the script, it will by default use the `--toy` option which will only use a very small subset of this. I will change this accordingly.